configs/cms-xcache/config.d/50-cms-xcache-paths.cfg

```diff
@@ -0,0 +1,22 @@
+#
+# Configure filesystem paths for the CMS XCache server.
+#
+# **********************************************************************
+# * WARNING: DO NOT EDIT THIS FILE.  IT WILL BE OVERWRITTEN ON UPGRADE *
+# **********************************************************************
+#
+# This file is part of the CMS XCache Daemon
+
+# The directory on local disk where files will be cached
+oss.localroot /xcache-root
+
+if exec xrootd
+   # The directories under oss.localroot to share data from.
+   all.export /store
+
+   # The directory on local disk where cinfo metadata files are held
+   oss.space meta /xcache-meta
```

Comment:
This should be originally set to same as $(rootdir) and let it up to the sites to change it

Reply:
Do you know if this is a variable that can be overriden if specified later in the config? The `40-` and `50-` prefixes are config files that we will write over on updates

Reply:
I do not know if not it is a bug to the Xrootd folks

Reply:
It looks like it appends rather than replaces:

```
Config continuing with file /etc/xrootd/config.d/50-cms-xcache-paths.cfg ...
=====> oss.localroot /mnt/stash 
=====> all.export /store
=====> oss.space meta /mnt/stash/xcache-meta
...
Config continuing with file /etc/xrootd/config.d/99-local-test.cfg ...
=====> oss.space meta /mnt/stash/new-xcache-meta
...
Config effective /etc/xrootd/xrootd-cms-xcache.cfg oss configuration:
...
       oss.space meta /mnt/stash/xcache-meta
       oss.space data /data1/xcache
       oss.space meta /mnt/stash/new-xcache-meta
```

Reply:
@efajardo do you know when metadata files are actually created?

Reply:
I talked to @osschar about this indeed you can have the metada sparsed on as many volumes you want and if not is setup I think it uses localroot. So what I would do is just totally remove that and leave it only to advanced users who want to separate data and metadata in different partitions. @brianhlin

Reply:
@efajardo this should be addressed in https://github.com/opensciencegrid/xcache/pull/64/commits/630735ad44325063b4fe879b999029fe631e623e

Reply:
Great this looks good to me